tools/generate_from_protos.sh

```diff
@@ -1,14 +1,15 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
That's necessary, because otherwise `script_dir` fails (`BASH_SOURCE[0]`, actually).